pandas/errors/__init__.py

```diff
@@ -65,3 +65,11 @@ class MergeError(ValueError):
     Error raised when problems arise during merging due to problems
     with input data. Subclass of `ValueError`.
     """
+
+
+class NullFrequencyError(ValueError):
+    """
```

Comment:
this needs a line in the api changes section

Reply:
done.